.github/workflows/tests-integration-reusable.yml

```diff
@@ -22,17 +22,29 @@ on:
       lease-id:
         required: true
         type: string
+      lease-namespace:
+        required: false
+        type: string
+        default: 'tas-devex'
       gitRef:
         type: string
         default: ${{github.event.workflow_run.head_sha}}
-        
+      nodes:
+        type: string
+        default: "8"
```

Comment:
The old default is 12, can we make that the default again?

Reply:
```suggestion
        default: "12"
```